vulture_whitelist.py

```diff
@@ -16,11 +16,14 @@
 
 _JSONEncoder.default
 
+TaskResult.filter_reports
```

Comment:
Is it really needed? It is used in tests.

Reply:
volture does not find it in test, hence it is required here